ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -447,6 +447,8 @@ open class TableViewCell: UITableViewCell {
     private static func textAreaTrailingOffset(customAccessoryView: UIView?, customAccessoryViewExtendsToEdge: Bool, accessoryType: TableViewCellAccessoryType, paddingTrailing: CGFloat) -> CGFloat {
         let customAccessoryViewAreaWidth: CGFloat
         if let customAccessoryView = customAccessoryView {
+            // Trigger layout so we can have the frame calculated correctly at this point in time
+            customAccessoryView.layoutIfNeeded()
```

Comment:
should the caller call do this before calling the static function ?

Reply:
I thought about this, but there are multiple places where this is called.
So, I think we can avoid potential future bugs and consolidate the check in here.